src/sessions.js

```diff
@@ -14,6 +14,8 @@ let hasPush = false
 
 let syncRunning = false
 
+const store = storeFactory()
```

Comment:
Could it be problematic here that we are not using the same instance as in the main app?